lib/Optimizer/CodeGen/Passes.cpp

```diff
@@ -21,6 +21,7 @@ static void addAnyonPPipeline(OpPassManager &pm) {
   };
   BasisConversionPassOptions options;
   options.basis = basis;
+  options.disabledPatterns = {"R1ToU3"};
```

Comment:
`disabledPatterns` is an ArrayRef, and since the temporary vector is destroyed after the statement, this will cause erroneous behavior. https://gist.github.com/bmhowe23/e84a6ea82820bdae3a5ead11f1808610 is one possible alternative that seems to work when I did local testing.